src/App.tsx

```diff
@@ -17,18 +59,39 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                selectedFilter={selectedFilter}
+                setSelectedFilter={(select: string) => setSelectedFilter(select)}
```

Comment:
apply for similar cases as well
```suggestion
                setSelectedFilter={setSelectedFilter}
```